src/routes/crud.py

```diff
@@ -0,0 +1,134 @@
+from datetime import datetime, timedelta
+import select
+from fastapi import HTTPException, Response
+from sqlalchemy.ext.asyncio import AsyncSession
+from database.models import MovieModel as Movie
+from schemas.movies import MovieCreate, MovieUpdate
+
+
+async def get_movie(movie_id: int, db: AsyncSession):
+    res = db.execute(select(Movie).where(Movie.id == movie_id))
```

Comment:
It's important to await the result of `db.execute` to ensure the asynchronous operation completes before proceeding. Missing `await` can lead to unexpected behavior.

```suggestion
res = await db.execute(select(Movie).where(Movie.id == movie_id))
```